src/System.Windows.Extensions/tests/System/Media/SystemSoundsTests.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Media.Test
+{
+    public class SystemSoundsTests
+    {
+        [Fact]
+        public void Asterisk_Get_ReturnsExpected()
+        {
+            SystemSound sound = SystemSounds.Asterisk;
+            Assert.NotNull(sound);
+            Assert.Same(sound, SystemSounds.Asterisk);
```

Comment:
This is probably fine, but note that this requires that nothing else in the process be accessing the same SystemSounds property concurrently, as the property isn't guaranteed to return the same instance if there's a race to initialize it.